src/main/resources/default/templates/wondergem/scripts/autocomplete.html.pasta

```diff
@@ -46,16 +30,43 @@
             }
         };
 
+        /**
+         * Represents the input field of the autocomplete.
+         *
+         * Events on the input field like onChange are defined here.
+         *
+         * @@type {{name: string, element: undefined, spinnerElement: undefined, spinnerTemplate: string, showSpinner: showSpinner, hideSpinner: hideSpinner, onChange: onChange, onNonInputKeys: onNonInputKeys, init: init}}
+         */
         var input = {
             name: "autocomplete-input",
             element: undefined,
+            spinnerElement: undefined,
```

Comment:
Do you really want to set these to undefined? I feel like it would be more correct to set them to null